django/db/backends/oracle/operations.py

```diff
@@ -93,16 +93,10 @@ def date_extract_sql(self, lookup_type, field_name):
 
     def date_interval_sql(self, timedelta):
         """
-        Implements the interval functionality for expressions
-        format for Oracle:
-        INTERVAL '3 00:03:20.000000' DAY(1) TO SECOND(6)
+        Implements the interval functionality.
```

Comment:
I guess this line isn't adding much value since we wouldn't put it on every date_interval_sql docstring.